test/VariantMatrixTest.cc

```diff
@@ -363,21 +363,16 @@ BOOST_AUTO_TEST_CASE(tests_windows_size_1)
         {
```

Comment:
A better test is to take a large data set from msprime and embed it as a string.  We can then take many windows/slices and compare to the input data.

Reply:
Done.